topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md

```diff
@@ -495,15 +484,14 @@ To share a history, click on the {% icon galaxy-gear %} icon in the history pane
 > Be sure to select **Also make all objects within the History accessible** whenever you make a history accessible via link, otherwise whomever you send your link to might not be able to see your history.
 {: .comment}
 
-> ### {% icon hands_on %} Hands-on: Share history and workflow
+> ### {% icon hands_on %} Hands-on: Share history
 >
-> 1. Share one of your histories with your neighbour.
-> 2. See if you can do the same with your workflow!
-> 3. Find the history and/or workflow shared by your neighbour. Histories shared with specific users can be accessed by those users in their {% icon galaxy-gear %} history menu under `Histories shared with me`.
+> 1. Share your history with your neighbour.
+> 2. Find the history shared by your neighbour. Histories shared with specific users can be accessed by those users in their {% icon galaxy-gear %} history menu under `Histories shared with me`.
```

Comment:
The location of "Histories Shared with me" has been removed from the History gear menu and is now available (only) under the top masthead "User" menu.

This is new in the `19.09` release (pending finalization). To review the updated GUI menus, the pre-release is loaded at https://usegalaxy.org

Reply:
Thanks. I updated tutorial.md accordingly.